src/Feature/Changelog/Feature_Changelog.re

```diff
@@ -293,10 +539,12 @@ module View = {
   module Update = {
     module Parts = {
       let line = (~commit, ~uiFont, ~theme, ()) => {
+        let splitSummary = commit.summary |> String.split_on_char('\n');
+
         <View style=Styles.commit>
           <typ commit uiFont theme />
           <scope commit uiFont theme />
-          <summary commit uiFont theme />
+          <title text={splitSummary |> List.hd} uiFont theme />
```

Comment:
Is there any chance `List.hd` could throw here? If so - we might want to protect against it (otherwise it could manifest as a crash-on-launch)

Reply:
Weird, I thought I pushed a change to fix that. One sec!

Reply:
Oh I forgot that that appeared twice -- I fixed it the first time but not the second.

Reply:
Awesome, thanks Zach!